web/content/docs/userguide/process-dependent-configuration/Heat_Transport_BHE_PipelineNetwork.pandoc

```diff
@@ -0,0 +1,278 @@
++++
+date = "2020-02-03T12:00:13+01:00"
+title = "Heat_Transport_BHE_PipeNetwork_feature"
+author = "Shuang Chen, Haibing Shao"
+weight = 1
+
+[menu]
+  [menu.userguide]
+    parent = "process-dependent-configuration"
++++
+
+## Introduction of the Pipe Network feature for the process Heat_Transport_BHE
+
+In a large borehole heat exchanger (BHE) array, all BHEs are connected with each other through a pipeline network. As the circulating fluid temperatures within each BHE are controlled by the network, the array itself has an intrinsic feature of balancing thermal extraction rates among different BHEs. This leads to the BHE thermal load shifting phenomena in the long-term operation. In OGS-6, this process can be simulated by using the PipeNetwork feature in the `Heat_Transport_BHE` process.
+
+This short tutorial aims to give the user a guide to the PipeNetwork feature. It includes the following main parts:\
+* The basic requirements for the PipeNetwork feature.\
+* The workflow of creating a pipeline network model using the software TESPy.\
+* The workflow of connecting the TESPy pipeline network model with the OGS model.\
+
+
+## Requirements on the computer
+
+In order to use the PipeNetwork feature, the computer running the `Heat_Transport_BHE` process of OpenGeoSys is required to have a Python 3 environment with the TESPy program installed. For the Python 3 environmental installation, there are various tutorial availabe on the Internet. For the TESPy package, one can install it with the following command line in the Python3 environment:
+
+```bash
+pip3 install tespy
+```
+
+## Creating a pipeline network model with the software TESPy
+
+The Thermal Engineering Systems in Python (TESPy (https://doi.org/10.5281/zenodo.2555866)) is a software package developed by Francesco Witte. It is capable of simulating coupled thermal-hydraulic status of working fluids in thermal engineering applications. Such systm typically involves a cirulation network that is composed of pre-defined components including pipes, heat exchangers and different types of turbo machinery. Interested readers may refer to the online documentation (https://tespy.readthedocs.io/en/master/index.html#) of TESPy for more detailed introduction of the software. The workflow in this part is built based on the benchmark example "A 3-BHE Array Coupled With Pipe Network" from the OpenGeoSys Documentation. One can refer to this benchmark and download its latest benchmark files from GitHub (https://www.opengeosys.org/docs/userguide/basics/introduction/) for creating the pipeline network TESPy model. The OpenGeoSys and TESPy version used for this tutorial is (OpenGeoSys 6.2.2) and 0.2.0 accordingly.
```

Comment:
Please make the hyperlink of TESPy embedded.

Reply:
Also embed a hyperlink to the benchmark "A 3-BHE Array Coupled With Pipe Network”.

Reply:
Modified in 934ba904116c329b7b913b636acaadcc9e137b11.